docs/src/getting-started/basics.md

```diff
@@ -43,8 +43,8 @@ abigen!(
     "your_project/out/debug/contract_test-abi.json",
 );
 
-// This helper will setup a local node/provider and a test wallet.
-let (provider, wallet) = setup_test_provider_and_wallet().await;
+// This helper will setup a test wallet linked to a local node/provider.
```

Comment:
`will launch a local node and provide a test wallet linked to it`, I think it's important to mention that a local node gets launched by the function.

Reply:
I found it redundant since it is already stated at the beginning, but I guess it can't hurt to repeat it